pages/linux/apt-key.md

```diff
@@ -1,6 +1,8 @@
 # apt-key
 
 > Key management utility for the APT Package Manager on Debian and Ubuntu.
+> Note: `apt-key` is now deprecated (except for the use of `apt-key del`).
```

Comment:
I had an old suggestion that suggested this, it it got erased and then I made a new one but forgot to add this:
```suggestion
> Note: `apt-key` is now deprecated (except for the use of `apt-key del` in maintainer scripts).
```